developer-guide.md

```diff
@@ -67,13 +83,30 @@ After cloning the repository open in in IntelliJ, there's some configuration req
 ![localhost-8080](./developer-guide-images/localhost-8080.png)
 
 
-## IntelliJ Tips ##
+### IntelliJ Tips ###
 
 Select Autoscroll from Source, to help you see where files are as you navigate search
 
 ![intellij auto scroll to file](./developer-guide-images/autoscroll.png)
 
+## Configure using Gradle ##
+
+To run a Jenkins instance containing a version of the plugin built directly from local source you can execute the following command from the root of the project directory
+> gradlew server
+
+Once running the Jenkins instance will be available at `http://127.0.0.1:8080` by default
+
+### Using Intellij ###
+
+1. Import project into Intellij as a Gradle project
+2. Ensure the correct Java version is set
+3. Run the `server` task from the Gradle panel
+
+![gradle-panel](./developer-guide-images/gradle-panel.png)
```

Comment:
We shouldn't need to teach people how to use Intellij in this document - but we should mention the tasks which are available (eg 'server').